mne/tests/test_coreg.py

```diff
@@ -1,7 +1,8 @@
 from glob import glob
 import os
+from shutil import copyfile
 
-from nose.tools import assert_equal, assert_raises, assert_true
+from nose.tools import assert_equal, assert_raises, assert_true, assert_is_not
```

Comment:
FYI we have gone to pytest so you can just do `assert x is not y` in tests now

Reply:
Do you want me to update the tests in this PR?

Reply:
Only if you are bored :)